gallerypage.py

```
@@ -1144,36 +1042,73 @@ def _write_jekyll(self, filepath, remote_keys, replacements,
                       self.release_date,
                       [date,                 ', '.join(other_dates)],
                       self.credit,
-                      '<a href="%s" target="_blank">%s</a>' % (self.origin_url,
-                                                               display_url),
+                      self.origin_url,
                       self.id,
             ]
 
             for (name, value) in zip(names, values):
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)